### PR TITLE
[switchbot_ros]force to use python3 for lambda string sort

### PR DESCRIPTION
**Pull request by mqcmd196**
Current code dies with python UnicodeEncodeError
```
[switchbot_ros-1] restarting process
process[switchbot_ros-1]: started with pid [9793]
[INFO] [WallTime: 1698028051.518458] [node:/switchbot_ros] [func:SwitchBotAction.get_switchbot_client]: Switchbot API Client initialized.
Traceback (most recent call last):
  File "/home/fetch/ros/melodic/src/jsk-ros-pkg/jsk_3rdparty/switchbot_ros/scripts/switchbot_ros_server.py", line 121, in <module>
    server = SwitchBotAction()
  File "/home/fetch/ros/melodic/src/jsk-ros-pkg/jsk_3rdparty/switchbot_ros/scripts/switchbot_ros_server.py", line 31, in __init__
    self.print_devices()
  File "/home/fetch/ros/melodic/src/jsk-ros-pkg/jsk_3rdparty/switchbot_ros/scripts/switchbot_ros_server.py", line 66, in print_devices
    key=lambda device: str(device['deviceName']))
  File "/home/fetch/ros/melodic/src/jsk-ros-pkg/jsk_3rdparty/switchbot_ros/scripts/switchbot_ros_server.py", line 66, in <lambda>
    key=lambda device: str(device['deviceName']))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-3: ordinal not in range(128)
[switchbot_ros-1] process has died [pid 9793, exit code 1, cmd /home/fetch/ros/melodic/src/jsk-ros-pkg/jsk_3rdparty/switchbot_ros/scripts/switchbot_ros_server.py __name:=switchbot_ros __log:=/home/fetch/.ros/log/cb16f396-7146-11ee-827c-c9d2ce386b33/switchbot_ros-1.log].
log file: /home/fetch/.ros/log/cb16f396-7146-11ee-827c-c9d2ce386b33/switchbot_ros-1*.log
```
This PR forces to use python3

### CODE REVIEW

**Comment by mqcmd196** [comment]
The PR is tested on real robot (Fetch1075: melodic) and my PC (noetic)

**Comment by mqcmd196** [comment]
If this PR is merged, I will handle https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/491

File: switchbot_ros/package.xml [hunk 1]
```diff
@@ -28,5 +27,6 @@
   <build_export_depend>actionlib_msgs</build_export_depend>
 
   <export>
+    requirements.txt
```
Comment (sktometometo) [inline switchbot_ros/package.xml:30]: We have to have `<pip_requrements>` tag here.
https://github.com/locusrobotics/catkin_virtualenv/blob/4af99703e6c7faf12c5d328dbab346250fd085f7/catkin_virtualenv/src/catkin_virtualenv/collect_requirements.py#L29C43-L38

```suggestion
    <pip_requirements>requirements.txt</pip_requirements>
```

**Review by sktometometo — CHANGES_REQUESTED** [review]

**Comment by sktometometo** [comment]
Great PR! I have left a comment.

**Review by sktometometo — APPROVED** [review]

File: switchbot_ros/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (tkmtnt7000) [inline switchbot_ros/requirements.txt:1]: This is my just curiosity.
Why is empty requirements.txt recommended? For future developers or catkin_virtualenv requirements?

File: switchbot_ros/CMakeLists.txt [hunk 3]
```diff
@@ -27,6 +28,10 @@ generate_messages(
 
 catkin_package()
 
+catkin_generate_virtualenv(
+  PYTHON_INTERPRETER python3
```
Comment (tkmtnt7000) [inline switchbot_ros/CMakeLists.txt:32]: Perhaps you need 
```
CHECK_VENV FALSE
```
for passing test.

File: switchbot_ros/requirements.txt (hunk 2 above)
Comment (mqcmd196) [reply switchbot_ros/requirements.txt]: I thought there is no `requirements.txt`, the catkin_virtualenv fails to create python environment, but it seems it doesn't need to.

File: switchbot_ros/CMakeLists.txt (hunk 3 above)
Comment (mqcmd196) [reply switchbot_ros/CMakeLists.txt:32]: Yes, I forgot that

File: switchbot_ros/requirements.txt (hunk 2 above)
Comment (tkmtnt7000) [reply switchbot_ros/requirements.txt]: Thank you. I see.

**Review by tkmtnt7000 — APPROVED** [review]
Sorry for late. LGTM.

**Pull request merged**